### DISCUSSION TITLE
Issues with virtualScrollerOptions in the DataTable component

### DISCUSSION THREAD

**Discussion by kzaruk** [primefaces/community#1335] [PrimeReact] [answered]
I want to create a table where new data is loaded when scrolling down, and I am using the virtualScrollerOptions parameter for this. The problem is that the vertical scroll is not working. 
Please, advise me on what I am doing wrong?

`      <DataTable
        virtualScrollerOptions={{
          lazy: true,
          onLazyLoad: loadCarsLazy,
          itemSize: 10,
          showLoader: true,
          loading: nnewTable.isLoading,
          loadingTemplate,
        }}
        value={nnewTable.items}
        rowGroupMode="rowspan"
        //@ts-ignore
        groupRowsBy={['nameLvl1', 'nameLvl2', 'nameLvl3']}
        sortMode="single"
        sortOrder={1}
        tableStyle={{ minWidth: '50rem' }}
        showGridlines
        headerColumnGroup={headerGroup}
        scrollable
        scrollHeight="800px"
      >
        {getTableChildCols()}
      </DataTable>`

`  const loadCarsLazy = async (event) => {
    if (!event.last) return;
    nnewTable.pagination.setPage(nnewTable.pagination.number + 1);
    await nnewTable.load();
  };`

`  const loadingTemplate = (options) => {
    return (
      <div className="flex align-items-center" style={{ height: '17px', flexGrow: '1', overflow: 'hidden' }}>
        <Skeleton width={options.cellEven ? (options.field === 'year' ? '30%' : '40%') : '60%'} height="1rem" />
      </div>
    );
  };`

**Comment by melloware** [comment, marked as answer]
Fully working example. Compare with this to see what you are doing wrong: https://stackblitz.com/edit/9mv6op?file=src%2FApp.jsx
